dashboard/app/models/level.rb

```diff
@@ -216,7 +216,9 @@ def delete_custom_level_file
     ['Unplugged', # no solutions
      'TextMatch', 'Multi', 'External', 'Match', 'ContractMatch', 'LevelGroup', # dsl defined, covered in dsl
      'Applab', 'Gamelab', # all applab and gamelab are freeplay
-     'NetSim', 'Odometer', 'Vigenere', 'FrequencyAnalysis', 'TextCompression', 'Pixelation'] # widgets
+     'EvaluationQuestion', # plc evaluation
+     'NetSim', 'Odometer', 'Vigenere', 'FrequencyAnalysis', 'TextCompression', 'Pixelation',
+    ] # widgets
```

Comment:
This list makes me sad 😞 .  It's also missing `FreeResponse` and maybe others.

Why bother with single-table-inheritance when you can have a list of strings in the base model! /s